nwbinspector/nwbinspector.py

```diff
@@ -117,6 +124,24 @@ def inspect_all(
     return organized_results
 
 
+def configure_checks(config, available_checks):
+    output_checks = copy.copy(available_checks)
+    for importance_name, func_names in config.items():
+        for func_name in func_names:
+            for importance, functions in output_checks.items():
+                if importance.name == importance_name:
+                    continue
+                i = 0
+                while i < len(functions):
+                    if functions[i].__name__ == func_name:
+
+                        output_checks[Importance._member_map_[importance_name]].append(functions.pop(i))
+                    else:
```

Comment:
While I think more about the main iteration restructuring, a couple implementation questions: 

a) what's the logic behind making a copy of the registry instead of acting directly on the users instance of `available_checks`?
b) note that this does not alter the internal `importance` attribute set within each check function; i.e., `functions[i].importance` - is this intended to remain at its default value both in the new copy and in the global available_checks?
c) For `Importance._member_map_[importance_name]` I'm pretty sure you can just do `Importance[importance_name]`, right?

Reply:
a) good point
b) yes, also good point. That's much better syntax.